examples/objective-c/route_guide/ViewControllers.m

```diff
@@ -83,17 +83,24 @@ + (instancetype)noteWithMessage:(NSString *)message
 @interface GetFeatureViewController : UIViewController {
   RTGRouteGuide *service;
```

Comment:
I missed this when reviewing #6440: ivars should be declared in the `@implementation`, and named with a `_` prefix (per https://google.github.io/styleguide/objcguide.xml#Instance_Variables_In_Headers_Should_Be_@private ). I think the best here for clarity would be to just turn it into a `@property`.